app/page.tsx

```diff
@@ -2,14 +2,12 @@
 
 import Link from 'next/link';
 
-import useUserStore from '@/stores/useUser.store';
-import { useAuth } from '@/hooks/useAuth';
 import Container from '@/components/layout/Container';
 import { Button } from '@/components/ui/button';
+import useUser from '@/hooks/useUser';
 
 export default function LandingPage() {
-  const { isAuthenticated } = useAuth();
-  const { user } = useUserStore();
+  const { user, isAuthenticated } = useUser();
 
```

Comment:
memberships를 추가해 :15 에 조건을 수정해야 할 것 같습니다
```suggestion
  const { user, isAuthenticated, memberships } = useUser();

  // :15줄 수정
  if (user && memberships?.length === 0) {
    ...
```

Reply:
수정했습니다!